background.gd

```diff
@@ -80,7 +80,10 @@ func _ready():
 	$Player.connect("land", _on_land)
 	#self.connect("foodcoincollision", _on_foodcoincollision)
 	self.connect("characterSelect", _on_characterSelect)
-	
+	LeaderboardsClient.score_submitted.connect(
+		func refresh_score(is_submitted: bool, leaderboard_id: String):
+			game_over()
```

Comment:
game_over() method is called, but leaderboard is not updated (despite successful login and user can check the achievements associated with their account as well)

Reply:
thats because i was using variable score, not points to update the leaderboard... whoops

Reply:
i ended up printing the is_submitted status, leaderboard_id to logs and noticed that everything matched as expected.. there had to be something else going on. 

yes , was passing 0 as score every time